content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -148,10 +142,13 @@ $ kubectl apply -f <(istioctl kube-inject \
 
 ## Migrating per-service mutual TLS enablement via annotations to authentication policy
 
-If you use service annotations to override global mutual TLS enablement for a service, you need to replace it with
-[authentication policy](/docs/concepts/security/#authentication-policies) and [destination rules](/docs/concepts/traffic-management/routing/destination-rules/).
+If you use service annotations to override global mutual TLS enablement for a
```

Comment:
@ security experts: is this still relevant today? It seemed to me this was a 1.0 -> 1.1 migration that was required, so it no longer needs to be documented. Is this the case?

Reply:
I'll leave this section to @yangminzhu to rework, and then rebase my PR on top.

Reply:
Agreed, this should not be needed from 1.1 to 1.2.